website/content/docs/internals/plugins/plugin-architecture.mdx

```diff
@@ -144,16 +123,33 @@ configured in the plugin catalog. Finally Vault runs the command configured in
 the catalog, sending along the JWT formatted response wrapping token and mlock
 settings. Like Vault, plugins support [the use of mlock when available](/docs/configuration#disable_mlock).
 
-~> Note: If Vault is configured with `mlock` enabled, then the Vault executable and each
-plugin executable in your [plugins directory](/docs/internals/plugins#plugin-directory) must be
-given the ability to use the `mlock` syscall.
+~> Note: If Vault is configured with `mlock` enabled, then the Vault executable
+and each plugin executable in your [plugins directory](/docs/internals/plugins/plugin-architecture#plugin-directory)
+must be given the ability to use the `mlock` syscall.
 
 ### Plugin Upgrades
 
-Plugins may be updated by registering and reloading them. More details on the
-upgrade procedure can be found in [Upgrading Vault Plugins](/docs/upgrading/plugins).
+External plugins may be updated by registering and reloading them. More details
+on the upgrade procedure can be found in
+[Upgrading Vault Plugins](/docs/upgrading/plugins).
+
+## Plugin Multiplexing
+
+As of Vault 1.10, database plugins can be made to implement plugin multiplexing
```

Comment:
Since we now have versioned docs, we no longer need to preface a release, i.e., As of Vault 1.10.